Makefile

```diff
@@ -22,6 +22,9 @@ update-deps:
 compile:
 	$(WEBPACK) -p
 
+compile-dev:
+	$(WEBPACK)
+
```

Comment:
What is the difference between `./node_modules/.bin/webpack` and `make compile-dev` really? I don't think the modification of the Makefile is integral to this PR. Please remove the modification.